cmd/gocharm/gocharm.go

```diff
@@ -196,6 +199,17 @@ func (b *charmBuilder) hookStub(hookName string) []byte {
 	})
 }
 
+func (b *charmBuilder) writeMetrics(metrics map[string]charm.Metric) error {
```

Comment:
I don't think this should write a metrics file if no metrics have been registered.